### PR TITLE
updated translation

### PR DESCRIPTION
**Pull request by banda-larga**
Left the term fine-tuning since there is no correct translation into Italian and the English term is generally used. The same was done with some terms like "learning rate"

### CODE REVIEW

**Review by sgugger — COMMENTED** [review]
Just pinging the original author @nickprock here to check if those changes are okay since none of us in the team actually speak Italian :-)

**Comment by nickprock** [comment]
Hi @banda-larga @sgugger we discuss a lot with @mfumanelli for the better italian translation. For me this version is ok

**Review by sgugger — APPROVED** [review]
Thanks for checking @nickprock and thanks a lot for the fixes @banda-larga !
